### DISCUSSION TITLE
Could I get the statecharts.io/inspect source code and deploy by myself

### DISCUSSION THREAD

**Discussion by SingleMai** [statelyai/xstate#2799] [Q&A] [answered]
**Expected**
I want get the source code of statecharts.io/inspect and try to deploy the site in the Intranet. I haven't found the git repository by the way.

** Reason **
First of all, I must thank you for providing such an excellent thing. However, in my case is hard to use it. Because my application had to use the "shardArrayBuffer" API in some reason. Thus chrome will force to use Cross-Origin-Opener-Policy   which will not allow to connect the child website when it's url not the same origin. 
[Link](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Opener-Policy)
> If a cross-origin document with COOP is opened in a new window, the opening document will not have a reference to it, and the window.opener property of the new window will be null. This allows you to have more control over references to a window than rel=noopener, which only affects outgoing navigations.

In order to fix it, I have to do the proxy thing to treat chrome for now. However if I can deploy by myself, it will be the best way to handle this problem.

**Comment by mattpocock** [comment, marked as answer]
@SingleMai The new version, xstate-viz, is here:

https://github.com/statelyai/xstate-viz

**Reply by SingleMai** [reply]
Thank you a lot, It's my fault to miss the README when I found it before. I will try it later.
